tools/ci_build/github/azure-pipelines/binary-size-checks-pipeline.yml

```diff
@@ -46,10 +54,35 @@ jobs:
       DockerBuildArgs: "--build-arg BUILD_UID=$( id -u )"
       Repository: onnxruntimecpubuild
 
-  - task: CmdLine@2
+  - task: PythonScript@0
+    displayName: 'Set variables from config file "$(BuildConfigFile)"'
+    inputs:
+      scriptSource: inline
+      script: |
+        import json
```

Comment:
Generally, I suggest putting such things in a separate python file so that our python lint could help.